src/hooks/auth/useAuthCheck.ts

```
@@ -18,6 +19,7 @@ export const useAuthCheck = () => {
             navigate(url, { replace: true });
         },
         authCheck: storage ? true : false,
-        playlistCode : storage?.playlistCode,
+        playlistCode : storage?.playlistCode!,
```

ESLint

[error] Optional chain expressions can return undefined by design - using a non-null assertion is unsafe and wrong.

(@typescript-eslint/no-non-null-asserted-optional-chain)